frontend/packages/shared/src/navigation/main-header/ProfileMenu/ProfileMenu.tsx

```diff
@@ -122,7 +119,7 @@ export const ProfileMenu = ({
           </a>
         </MenuItem>
         {showlogout && (
-          <MenuItem onClick={handleLogout} className={classes.menuItem}>
+          <MenuItem onClick={() => logout()} className={classes.menuItem}>
```

Comment:
This test is not covered as this file will be deleted [in the big PR that is being split into smaller chunks.](https://github.com/Altinn/altinn-studio/pull/13365)